### PR TITLE
Enable filtering on handler, last_error and queue

### PR DESCRIPTION
**Pull request by DanielZwijnenburg**
This PR enables users to search jobs based on handler, last_error and queue. I also renamed @queues to @filters for the sake of clarity.

### CODE REVIEW

**Comment by andyatkinson** [comment]
@DanielZwijnenburg Sorry about the delay. Are you still interested in getting this merged in? If so could you rebase and make any updates that are necessary? Are there any tests to add to help support the change? Thanks.

**Comment by DanielZwijnenburg** [comment]
I am, will write some specs later this week!

**Comment by andyatkinson** [comment]
@DanielZwijnenburg Bump

**Comment by DanielZwijnenburg** [comment]
@andyatkinson it has been a long time, just synced and rebased the latest changes. What is the best way to run this locally en test things?

**Comment by andyatkinson** [comment]
@DanielZwijnenburg Hello. Well, I'm not sure. Maybe in lieu of some automated integration testing, if you could just exercise your changes, do some filtering, searching or whatever, and add a couple of sentences describing what you did to test, that would be adequate. That way we could put that description into the release notes, in case it introduces a break. We could also create a new file to capture the manual testing steps. Thanks.

**Comment by DanielZwijnenburg** [comment]
@andyatkinson let me rephrase my question, how can I run this without mounting it in a rails app. I want to see my work in the browser and maybe add some more features.

**Comment by andyatkinson** [comment]
@DanielZwijnenburg Closing this due to inactivity. If this is something you're still interested in, please re-open this PR.
